app/models/user.rb

```diff
@@ -134,9 +134,9 @@ def registration_facility_id
   alias_method :facility, :registration_facility
 
   def full_teleconsultation_phone_number
-    defaulted_teleconsult_number = teleconsultation_phone_number.presence || phone_number
-    teleconsultation_isd_code ||= Rails.application.config.country["sms_country_code"]
```

Comment:
We had a bug here. Existing ISD codes were not respected because this statement initialized a `teleconsultation_isd_code` local variable, which would always be `nil` on the right side.

Reply:
What would have been the consequence of this? Numbers without ISD codes would've broken?

Reply:
Numbers _have_ to have ISD code. If all MOs added to IHCI had India phone numbers, then there would be no impact. If an MO had a non-india phone number, we'd clobber their ISD code with +91, and their number would be borked.

I think all MOs have India numbers, we should be good.